cmd/renterd/main.go

```diff
@@ -138,7 +140,8 @@ func getSeed() types.PrivateKey {
 	if seed == nil {
 		phrase := cfg.Seed
 		if phrase == "" {
-			fmt.Print("Enter seed: ")
+			fmt.Println("Type in your 12-word seed phrase and press enter. If you do not have a seed phrase yet, type 'seed' to generate one")
```

Comment:
`nit` I would inform the user he can generate a seed using the `renterd seed` command if he doesn't have one, but printing it like this I think some people would type seed here, and then be hit with `wrong number of words in seed phrase`

Reply:
They should be able to type “seed” and generate on if this is similar to `hostd`

Reply:
Oh, well that's fine too but then we should check for it because right now I think you'd be hit with `wrong number of words in seed phrase`. If that is not the case please feel free to resolve my comment, I haven't built it yet to try it out myself, but I will once this round of comments is done.

Reply:
You have to check whether the user's input equals "seed" because we've just instructed the user to type "seed" if he wants to generate one. You can look at [hostd](https://github.com/SiaFoundation/hostd/blob/master/cmd/hostd/main.go#L104) for inspiration.